cfme/configure/configuration/__init__.py

```diff
@@ -515,6 +515,47 @@ def update(self):
         self.appliance.server_details_changed()
 
 
+class VMwareConsoleSupport(Updateable, Pretty, Navigatable):
+    """
+    This class represents the "VMware Console Support" section of the Configuration page.
+    Note this is to support CFME 5.8 and beyond functionality.
+
+    Args:
+        console_type: VMware VMRC Plugin, VNC or VMware WebMKS
```

Comment:
**Required**
See the link below, which links to sphinxcontrib-napoleon, and add type information to this docstring for the args. Otherwise ambiguous if these are classes or literal strings

http://cfme-tests.readthedocs.io/guides/documenting.html?highlight=docstring